CONTRIBUTING.md

```diff
@@ -56,6 +56,30 @@ If you have any questions or need assistance, don't hesitate to reach out to us
 Happy Hacking!
 
 
+# Code Review Policy
+
+* CI must pass (enforced)
+* 1 review is mandatory (enforced), 2 or more ideal
+* If you believe your change is simple, and non-controversial enough, and you want
+  to avoid merge conflicts, or blocking work before it gets enough reviews, label it with
+  `Needs Further Review` tag and Merge it.
```

Comment:
Labels have a max length, I added [`Further Review`](https://github.com/fedimint/fedimint/labels/Further%20Review) instead.

Reply:
Duh, maybe it just collided and I didn't read the error https://github.com/fedimint/fedimint/labels/needs%20further%20review